modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/build.gradle

```diff
@@ -25,4 +25,5 @@ dependencies {
 	compileOnly project(":core:petra:petra-lang")
 	compileOnly project(":core:petra:petra-sql-dsl-api")
 	compileOnly project(":core:petra:petra-string")
+	testImplementation group: "junit", name: "junit", version: "4.13.1"
```

Comment:
@vngoliveira do we need this? see `EditRankingDisplayContextTest` they do not use it in their build.gradle, can we remove?